### PR TITLE
swaggersetup fix

### LINKED ISSUE #1853: Data Service Generator: add type to swaggerSetupOptions
In the file `swagger.template.ts` the swaggerSetupOptions const missing the type declaration `SwaggerCustomOptions`

### PR DESCRIPTION
**Pull request by Smartmind12**
Close #1853 

## PR Details

Add type to Swaggersetup options

## PR Checklist
- [X ] Tests for the changes have been added
- [X] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-data-service-generator/src/server/swagger/swagger.template.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const swaggerDocumentOptions = new DocumentBuilder()
   .AUTH_FUNCTION()
   .build();
 
-export const swaggerSetupOptions = {
+export const customOptions: SwaggerCustomOptions = {
```
Comment (abrl91) [inline packages/amplication-data-service-generator/src/server/swagger/swagger.template.ts:16]: Please don't rename the const, only add the type.

Comment (souravjain540) [reply packages/amplication-data-service-generator/src/server/swagger/swagger.template.ts:16]: any updates @Smartmind12 ?

**Review by tupe12334 — APPROVED** [review]

**Comment by tupe12334** [comment]
@yuval-hazaz Can you add @Smartmind12 to the `.all-contributorsrc`

**Comment by yuval-hazaz** [comment]
you can ask the bot :-) 

https://allcontributors.org/docs/en/bot/usage

**Comment by arielweinberger** [comment]
@all-contributors please add @Smartmind12 for feature enhancement.

**Comment by arielweinberger** [comment]
@all-contributors please add @Smartmind12 for feature maintenance.

**Pull request merged**